docs/UserGuide.md

```diff
@@ -5,13 +5,14 @@
 - [Quick Start](#quick-start)
 - [Usage](#usage)
 - [Features](#features)
-  1. [`add` - Add new items](#add-new-item)  
-  2. [`delete` - Delete item from list](#delete-an-item)
-  3. [`list` - List all items](#listing-all-items)
-  4. [`get` - Retrieve information of an item](#get-information-about-an-item)
-  5. [`edit` - Update an item](#edit-an-item)
-  6. [`report` - Generate sales report](#generate-sales-report)
-  7. [`bye` - Exit command](#exit-program)
+  1. [`help` - Show help](#display-help-message)
+  2. [`add` - Add new items](#add-new-item)  
+  3. [`delete` - Delete item from list](#delete-an-item)
+  4. [`list` - List all items](#listing-all-items)
+  5. [`get` - Retrieve information of an item](#get-information-about-an-item)
+  6. [`edit` - Update an item](#edit-an-item)
+  7. [`report` - Generate sales report](#generate-sales-report)
+  8. [`bye` - Exit command](#exit-program)
```

Comment:
I think it's better to create separate PR for general documentation updates.
Each PR is supposed to be only targeting one issue.